src/reactionsystem.jl

```diff
@@ -1495,7 +1520,7 @@ end
 # SDEProblem from AbstractReactionNetwork
 function DiffEqBase.SDEProblem(rs::ReactionSystem, u0, tspan,
                                p = DiffEqBase.NullParameters(), args...;
-                               noise_scaling = nothing, name = nameof(rs),
+                               noise_scaling = get_noise_scaling(rs), name = nameof(rs),
```

Comment:
I'd suggest we deprecate this, so by default it is `nothing`. If a user passes anything here and `get_noise_scaling` is empty we can use it, but give a warning that it is deprecated. Likewise, if a user already set `get_noise_scaling` and uses this we give an error that both can't be used. Then in a future breaking release we can drop this kwarg, which is non-standard for `SDEProblem` anyway.

Reply:
Sounds good, yeah, I will do this.